components/RecipeList.js

```diff
@@ -1,17 +1,23 @@
-import data from "@/db/data";
 import Image from "next/image";
 import styled from "styled-components";
 import Link from "next/link";
 
-export default function RecipeList() {
+export default function RecipeList({ recipes }) {
+  //Add empty entry for the Create Recipe Link/Symbol
+  const recipeList = [...recipes];
```

Comment:
this is not adding an empty entry. It is creating a new array and filling it up with all the recipes you have in the recipes array you pass.

Why are you doing that?

Reply:
Yeah you are right we forgot to remove it. We used that to append a empty object. We overcomplicated the <li> entry for the add recipe (+) link at the end of the <ul>. 😅